Looms.PoS/Looms.PoS.Application/Features/Order/Queries/GetOrders/GetOrdersQuery.cs

```diff
@@ -0,0 +1,19 @@
+using Looms.PoS.Application.Abstracts;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Order.Queries.GetOrders;
+
+public record GetOrdersQuery : LoomsHttpRequest, IRequest<IActionResult>
+{
+    public string? Status { get; init; }
+    public string? UserId { get; init; }
```

Comment:
both these properties should not be nullable

Reply:
How so?

Reply:
because when we are working with them they should not be null and that is ensured by the validator

Reply:
But these query parameters aren't required, they're optional, thus I don't see why they should be not nullable.